src/Wallabag/CoreBundle/Resources/views/themes/material/Entry/_card_actions.html.twig

```diff
@@ -9,6 +9,7 @@
 
     <ul class="tools right">
         <li>
+            <a title="{{ 'entry.list.show_same_domain'|trans }}" class="tool grey-text" href="{{ path('same_domain', { 'id': entry.id }) }}"><i class="material-icons">language</i></a>
```

Comment:
That translation is missing in the translation file.

Reply:
Fixed. Good catch.